docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -1134,7 +1124,55 @@ You can now run the `image-builder` CLI with the `files-config` option, with thi
    ```bash
    image-builder build --os <OS> --hypervisor <hypervisor> --release-channel <release channel> --<hypervisor>-config config.json --files-config files.json
    ```
+### Using Proxy Server
 
+`image-builder` supports proxy enabled build environments. In order to use proxy server to route outbound requests to internet, add the following fields to the hypervisor or provider configuration file (e.x. `baremetal.json`)
+
+  ```json
+   {
+     "http_proxy": "<http proxy endpoint, for example, http://username:passwd@proxyhost:port",
+     "https_proxy": "<https proxy endpoint, for example, https://proxyhost:port/",
+     "no_proxy": "<optional comma seperated list of domains that should be excluded from proxying"
+  }
+  ```
+
+Run `image-builder` CLI with the hypervisor configuration file
+  ```bash
+  image-builder build --os <OS> --hypervisor <hypervisor> --release-channel <release channel> --<hypervisor>-config config.json
+  ```
+
+### RedHat Satellite Support
+
+As part of the image building process, `image-builder` creates a virtual machine with the chosen operating system, install all the required packages and creates an image out of the virtual machine.
+While building RedHat node images, `image-builder` uses public RedHat subscription endpoints to register the build virtual machine with the provided RedHat account and download required packages.
+
+Alternatively, `image-builder` can also use a private RedHat Satellite to register the build virtual machine and pull packages from the Satellite. 
+In order to use RedHat Satellite in the image build process follow the steps below.
+
+#### Prerequisites
+1. Ensure the host running `image-builder` has bi-directional network connectivity with the RedHat Satellite
+2. Image builder flow only supports RedHat Satellite version >= 6.8
+3. Add the following RedHat repositories for the latest RedHat 8.x version on the Satellite and initiate a sync to replicate required packages
```

Comment:
Nutanix added support for RHEL 9. So this can also be 9.x if building Nutanix RHEL 9 images right?